src/fidesops/service/connectors/query_config.py

```diff
@@ -242,10 +261,15 @@ class SQLQueryConfig(QueryConfig[TextClause]):
 
     def format_fields_for_query(
         self,
-        fields: List[str],
+        field_paths: List[FieldPath],
     ) -> List[str]:
-        """Returns fields in a format they can be added into SQL queries."""
-        return fields
+        """Returns field pathspaths in a format they can be added into SQL queries.
```

Comment:
There's an extra `paths` in `pathspaths`

Reply:
good spot